Error traces downloading favicons

### Subject of the issue

I see this traces in my logs.
* The first traces I'm not sure if they should be at "error" level or just warning.
* The last trace is multi-line and it does not provide enough information about the domain/url.

```
[2021-12-06 22:46:48.736][request][INFO] GET /icons/192.168.1.1/icon.png,
[2021-12-06 22:46:48.737][vaultwarden::api::icons][WARN] IP 192.168.1.1 for domain '192.168.1.1' is not a global IP!,
[2021-12-06 22:46:48.737][vaultwarden::api::icons][ERROR] Error downloading icon: Domain is blacklisted. 192.168.1.1,

[2021-12-06 22:46:49.206][request][INFO] GET /icons/192.168.1.250/icon.png
[2021-12-06 22:46:49.206][vaultwarden::api::icons][WARN] IP 192.168.1.250 for domain '192.168.1.250' is not a global IP!,
[2021-12-06 22:46:49.206][vaultwarden::api::icons][ERROR] Error downloading icon: Domain is blacklisted. 192.168.1.250,

[2021-12-06 22:47:20.656][vaultwarden::api::icons][ERROR] Error downloading icon: Req.,
[CAUSE] reqwest::Error {,
    kind: Decode,,
    source: TimedOut,,
},
[2021-12-06 22:47:20.656][response][INFO] GET /icons/<domain>/icon.png (icon) => 200 OK
```

### Deployment environment

Official docker image:
vaultwarden/server:1.23.0-alpine

### Expected behaviour

I have a monitoring system with "error" alerts and I think those messages are not critical enough to be at "error" level.

--- a/src/api/icons.rs
+++ b/src/api/icons.rs
@@ -294,7 +294,7 @@ fn is_domain_blacklisted(domain: &str) -> bool {
 
             // Use the pre-generate Regex stored in a Lazy HashMap.
             if regex.is_match(domain) {
-                warn!("Blacklisted domain: {} matched ICON_BLACKLIST_REGEX", domain);
+                debug!("Blacklisted domain: {} matched ICON_BLACKLIST_REGEX", domain);
                 is_blacklisted = true;
             }
         }
@@ -330,7 +330,7 @@ fn get_icon(domain: &str) -> Option<(Vec<u8>, String)> {
             Some((icon, icon_type.unwrap_or("x-icon").to_string()))
         }
         Err(e) => {
-            error!("Error downloading icon: {:?}", e);
+            warn!("Unable to download icon: {:?}", e);
             let miss_indicator = path + ".miss";
             save_icon(&miss_indicator, &[]);
             None
@@ -599,7 +599,7 @@ fn get_page(url: &str) -> Result<Response, Error> {
 
 fn get_page_with_referer(url: &str, referer: &str) -> Result<Response, Error> {
     if is_domain_blacklisted(url::Url::parse(url).unwrap().host_str().unwrap_or_default()) {
-        err!("Favicon resolves to a blacklisted domain or IP!", url);
+        warn!("Favicon '{}' resolves to a blacklisted domain or IP!", url);
     }
 
     let mut client = CLIENT.get(url);
@@ -757,10 +757,10 @@ fn save_icon(path: &str, icon: &[u8]) {
             f.write_all(icon).expect("Error writing icon file");
         }
         Err(ref e) if e.kind() == std::io::ErrorKind::NotFound => {
-            create_dir_all(&CONFIG.icon_cache_folder()).expect("Error creating icon cache");
+            create_dir_all(&CONFIG.icon_cache_folder()).expect("Error creating icon cache folder");
         }
         Err(e) => {
-            warn!("Icon save error: {:?}", e);
+            warn!("Unable to save icon: {:?}", e);
         }
     }
 }

